tsconfig.json

```diff
@@ -2,7 +2,7 @@
   "compilerOptions": {
     /* Basic Options */
     // "incremental": true,                   /* Enable incremental compilation */
-    "target": "es5",                          /* Specify ECMAScript target version: 'ES3' (default), 'ES5', 'ES2015', 'ES2016', 'ES2017', 'ES2018', 'ES2019' or 'ESNEXT'. */
+    "target": "es2019",                          /* Specify ECMAScript target version: 'ES3' (default), 'ES5', 'ES2015', 'ES2016', 'ES2017', 'ES2018', 'ES2019' or 'ESNEXT'. */
```

Comment:
Can you clarify why we need es2019?

Reply:
Oops, I meant to change this. I was thinking for `build` folder it only needs to target whatever ES the node version we're supporting has, but for build, it needs to target whatever browser version (I guess that's ES5 default). Instead of splitting it, I'll just move it back to ES5.